app/src/androidTest/java/nextstep/signup/SignUpScreenTest.kt

```diff
@@ -29,59 +32,171 @@ class SignUpScreenTest {
     }
 
     @Test
-    fun signup_text_field_test() {
+    fun signup_button_test() {
         // given
         composeTestRule.setContent {
-            SignUpTextField(SignUpTextFieldType.USERNAME)
-            SignUpTextField(SignUpTextFieldType.EMAIL)
-            SignUpTextField(SignUpTextFieldType.PASSWORD)
-            SignUpTextField(SignUpTextFieldType.PASSWORD_CONFIRM)
+            SignUpButton()
+        }
+
+        // then
+        composeTestRule
+            .onNodeWithText("Sign Up")
+            .assertExists()
+    }
+
+    @Test
+    fun signup_screen_test() {
+        // given
+        composeTestRule.setContent {
+            SignUpScreen()
+        }
+
+        // then
+        composeTestRule
+            .onRoot()
+            .onChildren()
+            .assertCountEquals(6)
+            .onFirst()
+            .assert(hasText("Welcome to Compose \uD83D\uDE80"))
+    }
+
+    @Test
+    fun 유저이름_글자수_미달_오류_테스트() {
```

Comment:
```suggestion
    fun 유저이름_글자수_2자미만_오류_테스트() {
```
함수명만 보고 미달인 조건을 쉽게 인지할 수 있도록 변경해보면 어떨까요?
(이하 관련 내용 동일)

Reply:
다시 보니 테스트 함수명만 보고 인지하기 어려운 것 같네요.
더 쉽게 인지할 수 있도록 피드백 반영해서 수정하겠습니다!